qa/suites/crimson-rados/thrash/objectstore/seastore.yaml

```diff
@@ -0,0 +1 @@
+.qa/config/crimson_seastore.yaml
```

Comment:
@cyx1231st, note: previously we had `thrash` and `singleton` enabled only w/ Bluestore.
With this change we will run then with both backends (90 total tests instead of 78).

Reply:
To double confirm, is:
```
teuthology-suite -m smithi -e <mail> -p <priority> -s crimson-rados -c <branch> --filter-out perf,'objectstore/bluestore'
```
still valid to validate SeaStore-only changes ?

Reply:
Yes, nothing changed in the suite itself. I've only added `thrash` and `singleton` to Seastore as well.